libs/sdk-core/src/models.rs

```diff
@@ -833,9 +841,12 @@ impl OpeningFeeParamsMenu {
 
         // Of those, return the first, which is the cheapest
         // (sorting order of fee params list was checked when the menu was initialized)
-        valid_min_48h.first().cloned().ok_or(anyhow!(
-            "The LSP doesn't support opening fees with a validity > 48 hours"
-        ))
+        valid_min_48h
+            .first()
+            .cloned()
+            .ok_or_else(|| SdkError::LspOpenChannelNotSupported {
+                err: "The LSP doesn't support opening fees with a validity > 48 hours".into(),
```

Comment:
NIT: maybe:
`The LSP doesn't support opening fees that are valid for at least 48 hours`

Reply:
Done